src/config/index.ts

```diff
@@ -82,6 +82,11 @@ export const FEATURE_FLAG_ENABLE_AUTH = getEnvironmentValue('FEATURE_FLAG_ENABLE
 export const FEATURE_FLAG_ENABLE_NOTIFY = getEnvironmentValue('FEATURE_FLAG_ENABLE_NOTIFY', 'false');
 export const FEATURE_FLAG_ENABLE_DYNAMO = getEnvironmentValue('FEATURE_FLAG_ENABLE_DYNAMO', 'false');
 
+// Auth URLs
+export const AUTH_PATH = 'https://cola.service.cabinetoffice.gov.uk/v2/github-requests';
```

Comment:
The `AUTH_PATH` should be dynamic and fetched from the `envs`, we use to have it on our `env` named AUTH_SIGN_IN_URL as you can see [here](https://github.com/cabinetoffice/github-requests-app/blob/56a9fff70b7f4ede51acaa82dfddc0fa54a4fd5f/README.md?plain=1#L31)

Reply:
Yep you're right, forgot to import it